pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+mongodb_fixture_dir =
+  tests/mongodb/fixtures
```

Comment:
Add to appropriate section in `setup.cfg`:

https://github.com/r-bioinformatics/edgePy/blob/cf1691a2c48b7753e3bbd0d44981e196fe8cb7bf/setup.cfg#L59-L60

Reply:
After much experimentation, I can't get this to work.  Any hints?

I've tried 
`addopts = ....  --mongodb_fixture_dir tests/mongodb/fixtures`

and 

`[tool:pytest]
addopts = --verbose --doctest-modules --cov
mongodb_fixture_dir =
  tests/mongodb/fixtures`

among others..   no dice.